### PR TITLE
Fix incorrect terminate binding contract logic

### PR DESCRIPTION
**Pull request by meaboutsoftware**
This PR includes changes related to:

- searching for binding contract using its id instead of contract id
- fix of the API path for binding contracts
- returning binding contract id while signing the contract
- adjusting access modifiers to internal
- renaming

### CODE REVIEW

File: Chapter-4-applying-tactical-domain-driven-design/Fitnet.Contracts/Src/Fitnet.Contracts.Api/ContractsApiPaths.cs [hunk 1]
```diff
@@ -5,7 +5,7 @@ namespace EvolutionaryArchitecture.Fitnet.Contracts.Api;
 internal static class ContractsApiPaths
 {
     private const string ContractsRootApi = $"{ApiPaths.Root}/contracts";
-    private const string BindingContractsApi = $"{ContractsRootApi}/binding";
+    private const string BindingContractsApi = $"{ApiPaths.Root}/bindingContracts";
```
Comment (kamilbaczek) [inline Chapter-4-applying-tactical-domain-driven-design/Fitnet.Contracts/Src/Fitnet.Contracts.Api/ContractsApiPaths.cs:8]:
```csharp
private const string BindingContractsApi = $"{ApiPaths.Root}/binding-contracts";
```  

What you think?

File: Chapter-4-applying-tactical-domain-driven-design/Fitnet.Contracts/Src/Fitnet.Contracts.Api/SignContract/SignContractEndpoint.cs [hunk 2]
```diff
@@ -14,9 +14,9 @@ internal static class SignContractEndpoint
             IContractsModule contractsModule, CancellationToken cancellationToken) =>
         {
             var command = request.ToCommand(id);
-            await contractsModule.ExecuteCommandAsync(command, cancellationToken);
+            var bindingContractId = await contractsModule.ExecuteCommandAsync(command, cancellationToken);
 
-            return Results.NoContent();
+            return Results.Ok(bindingContractId);
```
Comment (kamilbaczek) [inline Chapter-4-applying-tactical-domain-driven-design/Fitnet.Contracts/Src/Fitnet.Contracts.Api/SignContract/SignContractEndpoint.cs:19]:
```csharp
return Results.Created($"/{ContractsApiPaths.BindingContracts}/{bindingContractId}", bindingContractId);
```

File: Chapter-4-applying-tactical-domain-driven-design/Fitnet.Contracts/Src/Fitnet.Contracts.Api/SignContract/SignContractEndpoint.cs [hunk 3]
```diff
@@ -25,7 +25,7 @@ internal static class SignContractEndpoint
             Description =
                 "This endpoint is used to sign prepared contract by customer.",
         })
-        .Produces(StatusCodes.Status204NoContent)
+        .Produces(StatusCodes.Status200OK)
```
Comment (kamilbaczek) [inline Chapter-4-applying-tactical-domain-driven-design/Fitnet.Contracts/Src/Fitnet.Contracts.Api/SignContract/SignContractEndpoint.cs:28]:
```csharp
StatusCodes.201CREATED
```

**Review by kamilbaczek — CHANGES_REQUESTED** [review]

**Comment by kamilbaczek** [comment]
Please update .http documentation as well according to changing API

File: Chapter-4-applying-tactical-domain-driven-design/Fitnet.Contracts/Src/Fitnet.Contracts.Api/ContractsApiPaths.cs (hunk 1 above)
Comment (meaboutsoftware) [reply Chapter-4-applying-tactical-domain-driven-design/Fitnet.Contracts/Src/Fitnet.Contracts.Api/ContractsApiPaths.cs]: Done

File: Chapter-4-applying-tactical-domain-driven-design/Fitnet.Contracts/Src/Fitnet.Contracts.Api/SignContract/SignContractEndpoint.cs (hunk 3 above)
Comment (meaboutsoftware) [reply Chapter-4-applying-tactical-domain-driven-design/Fitnet.Contracts/Src/Fitnet.Contracts.Api/SignContract/SignContractEndpoint.cs]: Done

File: Chapter-4-applying-tactical-domain-driven-design/Fitnet.Contracts/Src/Fitnet.Contracts.Api/SignContract/SignContractEndpoint.cs (hunk 2 above)
Comment (meaboutsoftware) [reply Chapter-4-applying-tactical-domain-driven-design/Fitnet.Contracts/Src/Fitnet.Contracts.Api/SignContract/SignContractEndpoint.cs]: Done

**Review by kamilbaczek — APPROVED** [review]
Great! Thanks 👍🏻

**Pull request merged**